src/server/game/Warden/Warden.h

```diff
@@ -113,10 +113,12 @@ class Warden
     virtual void InitializeModule() = 0;
     virtual void RequestHash() = 0;
     virtual void HandleHashResult(ByteBuffer &buff) = 0;
-    virtual uint16 RegisterPayload(std::string& payload) = 0;
+    virtual uint16 RegisterPayload(const std::string& payload) = 0;
```

Comment:
```suggestion
    virtual uint16 RegisterPayload(std::string const& payload) = 0;
```

Reply:
https://www.azerothcore.org/wiki/cpp-code-standards#defining-const-variables

Reply:
Ah my bad, I should probably look at that guide instead of just going off the code-base. Fixed :)